avocado/plugins/jobs.py

```diff
@@ -56,14 +56,25 @@ def _print_job_tests(tests):
         for test in tests:
             status = test.get("status")
             decorator = output.TEST_STATUS_DECORATOR_MAPPING.get(status)
-            # Retrieve "end" for backward compatibility
-            end = datetime.fromtimestamp(test.get("actual_end", test.get("end")))
+
+            end_timestamp = test.get("actual_end", test.get("end"))
+            if end_timestamp is not None:
+                end = datetime.fromtimestamp(end_timestamp).strftime(date_fmt)
+            else:
+                end = "N/A"
```

Comment:
In Avocado we use `<unknown>` instead of `N/A` in result values. For example, see [human-readable output](https://github.com/avocado-framework/avocado/blob/master/avocado/plugins/human.py#L124).